pkg/collector/collector.go

```diff
@@ -415,10 +417,18 @@ func (cm *CollectorManager) CollectContainerEvents(id *ContainerId) {
 		}
 
 		// The name of the ApplicationProfile you're looking for.
-		appProfileName := fmt.Sprintf("pod-%s", id.PodName)
+		var appProfileName string
+		var namespace string
+		if cm.config.StoreNamespace != "" {
+			appProfileName = fmt.Sprintf("pod-%s.%s", id.PodName, id.Namespace)
```

Comment:
Is this working? I thought you cannot have `.` in object names. I have check it [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/) and out of three standards, only two let you use it.
I know the delimitation is a problem

Reply:
So the only way to go around it is to put the original namespace in the annotations because I can't parse it from the name.

Reply:
done with labels